src/Browscap/Data/Expander.php

```diff
@@ -43,6 +43,11 @@ class Expander
     private $logger = null;
 
     /**
+     * @var array
```

Comment:
I could set it here to the empty state that is seen below (line 102). I didn't do that initially as I thought it was unnecessary duplication.